k9mail/src/main/java/com/fsck/k9/activity/MessageList.java

```diff
@@ -1435,6 +1435,8 @@ public void enableActionBarProgress(boolean enable) {
     public void displayMessageSubject(String subject) {
         if (mDisplayMode == DisplayMode.MESSAGE_VIEW) {
             mActionBarSubject.setText(subject);
```

Comment:
The call in the else block is only a trampoline to `MessageHeader.showSubjectLine()`. The `MessageHeader` instance already knows about the subject, so it's not passed in as an argument.